shaptools/hana.py

```diff
@@ -84,8 +84,6 @@ class HanaInstance(object):
         'x86_64', 'ppc64le'
     ]
     SUPPORTED_SYSTEMS = ['Linux']
-    # SID is usualy written uppercased, but the OS user is always created lower case.
-    HANAUSER = '{sid}adm'.lower()
```

Comment:
We are already setting the user to lowercase here. I guess this wasnt working in python? that it shows `PRDadm`

Reply:
Yeah... This stupid piece of work doesn't work XD The lower function is applied before the formatting.
And we didn't have any proper UT neither